l10n/en/firefox/whatsnew/whatsnew-104-default.ftl

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/104.0/whatsnew/
+
+whatsnew-104-default-title = Feel safe every time you explore the web
+whatsnew-104-default-body = { -brand-name-firefox } is rolling out its strongest privacy protection to date: Total Cookie Protection confines cookies to the sites where they originated, preventing tracking companies from using them to follow you as you browse from site to site. { -brand-name-firefox } will always advocate and innovate for your privacy.
```

Comment:
**Question**(non-blocking): should "Total Cookie Protection" have a comment here to explain that its one of our features? I wonder if it would be weird translated?

Reply:
I'm not sure if it's a real browser feature or just marketing lingo. We used the phrase on the previous page and it was translated pretty literally in French and German. @peiying2 might have advice on how we should treat this.